content/Foundation/Summit/2024/home.pml

```diff
@@ -0,0 +1,93 @@
+    SPDX-License-Identifier: CC-BY-SA-4.0
+-->
+<h1>
+  seL4 Summit 2024
+</h1>
+<h4>
+  Sydney, Australia
+</h4>
+<h4>
+  October 2024 (dates TBC)
+</h4>
+
+
+<div class="summit_div">
+  <ul>
+    <!--li><a href="#summit-important-dates">Important Dates</a></li-->
+    <li><a href="#summit-about">About</a></li>
+    <li><a href="../principles">Principles</a></li>
+    <!--li><a href="#summit-registration">Registration</a></li-->
+    <li><a class="highlight" href="#latest-news">Latest News: Location & Venue</a></li>
+    <!--li><a href="cfp">Call for Presentations</a></li>
+    <li><a href="#summit-keynote-speakers">Keynote Speakers</a></li>
+    <li><a href="#summit-panellists">Panellists</a></li>
+    <li><a href="#summit-program">Program, Abstracts, Slides and Videos</a></li>
+    <li><a href="#summit-pc">Program Committee</a></li>
+    <li><a href="#summit-sponsors">Sponsors</a></li>
+    <li><a href="#summit-contact">Contact</a></li-->
+  </ul>
+</div>
+
+<div class="summit_div">
+  <a href="../">All summits</a>
+</div>
+
+<!--div class="summit_div">
+  <h3 id="summit-important-dates">
+    Important Dates
+  </h3>
+  <ul>
+    <li><s>Abstracts due: </s></li>
+    <li><s>Notification of decisions: </s> </li>
+    <li><s>Early bird cut-off: </s></li>
+    <li><strong>Summit: </strong></li>
+  </ul>
+</div-->
+
+<div class="summit_div">
+  <h3 id="summit-about">
+    About
+  </h3>
+  <p>
+    The seL4 Summit is the annual international summit on the <a href="../../../About">seL4 microkernel</a>, the world's most highly assured OS kernel, as well as on all seL4-related technology, tools, infrastructure, products, projects, and people.
+  </p>
+  <p>
+    It aims to gather all the seL4 community to learn, share, and connect:
+  </p>
+    <ul>
+      <li>learn about the seL4 technology, its latest progress, use, successes, challenges, plans;</li>
+      <li>share exciting seL4 development, research, experience, application in the real world;</li>
+      <li>connect with other seL4 developers, users, providers, customers, supporters, potential partners and enthusiasts.</li>
+    </ul>
+  <p>
+    The seL4 Summit is organised by the <a href="../../../Foundation">seL4 Foundation</a>, in a location aimed to be on a different continent each year, as far as possible.
+  </p>
+  <p>
+    An open call for presentations invites submissions of short abstracts about cool work on seL4, and a Program Committee made of a wide range of representatives of the seL4 community is in charge of the technical content (reviewing submission, selecting invited speakers, defining the program).
+  </p>
+</div>
+
+
+<div class="summit_div">
+  <div class="highlight">
+    <h3 id="latest-news">
+      Location & Venue
+    </h3>
+  </div>
+  <p>
+    The seL4 Summit 2024 will be held in Sydney, Australia in October 2024 (date and venue TBC). The summit is being organised by the <a href="https://events.linuxfoundation.org/">Linux Foundation</a>.
```

Comment:
I realise we say "organised by the seL4 Foundation above" and "organised by the Linux Foundation" here. I think we don't need to mention LF until we have the extra page with venue info. Then we can add something like "The logistics and venue are being handled by.."